quickstart/README.adoc

```diff
@@ -1,45 +1,395 @@
+`local-datacenter`, which is required by the driver, and – optionally – the keyspace to bind to.
+
+A sample configuration should look like this:
+
+[source,properties]
+----
+quarkus.cassandra.contact-points={cassandra_ip}:9042
+quarkus.cassandra.local-datacenter={dc_name}
+quarkus.cassandra.keyspace={keyspace}
+----
+
+In this example, we are using a single instance running on localhost, and the keyspace containing
+our data is `k1`:
+
+[source,properties]
+----
+quarkus.cassandra.contact-points=127.0.0.1:9042
+quarkus.cassandra.local-datacenter=datacenter1
+quarkus.cassandra.keyspace=k1
+----
+
+If your cluster requires plain text authentication, you can also provide two more settings:
+`username` and `password`.
+
+[source,properties]
+----
+quarkus.cassandra.auth.username=john
+quarkus.cassandra.auth.password=s3cr3t
+----
+
+=== Connecting to a cloud DataStax Astra database
+
+When connecting to link:https://astra.datastax.com[DataStax Astra], instead of providing a contact
+point and a datacenter, you should provide a so-called _secure connect bundle_, which should point
+to a valid path to an Astra secure connect bundle file. You will also need to provide a username and
+password, since authentication is always required on Astra clusters.
+
+You can download your secure connect bundle from the Astra web console.
+
+A sample configuration for DataStax Astra should look like this:
+
+[source,properties]
+----
+quarkus.cassandra.cloud.secure-connect-bundle=/path/to/secure-connect-bundle.zip
+quarkus.cassandra.auth.username=john
+quarkus.cassandra.auth.password=s3cr3t
+quarkus.cassandra.keyspace=k1
+----
+
+=== Advanced driver configuration
+
+You can configure other Java driver settings using `application.conf` or `application.json` files.
+They need to be located in the classpath of your application. All settings will be passed
+automatically to the underlying driver configuration mechanism. Settings defined in
+`application.properties` with the `quarkus.cassandra` prefix will have priority over settings
+defined in `application.conf` or `application.json`.
+
+To see the full list of settings, please refer to the
+link:https://docs.datastax.com/en/developer/java-driver/latest/manual/core/configuration/reference/[driver
+settings reference].
+
+== Running a Cassandra Database
+
+By default, the Cassandra client is configured to access a local Cassandra database on port 9042
+(the default Cassandra port).
+
+IMPORTANT: Make sure that the setting `quarkus.cassandra.local-datacenter` matches the datacenter of
+your Cassandra cluster.
+
+TIP: If you don't know the name of your local datacenter, this value can be found by running the
+following CQL query: `SELECT data_center FROM system.local`.
+
+If you want to use Docker to run a Cassandra database, you can use the following command to launch
+one:
 
 [source,shell]
 ----
-docker run --name local-cassandra-instance cassandra
+docker run --name local-cassandra-instance -p 9042:9042 cassandra
```

Comment:
Might be worth suggesting running the container with the `-d` option to keep it in the background.  Mainly just because in the next step we suggest a few other docker comments to setup the DB, which means starting up a new console.